### ISSUE TITLE
Lint HTML embedded in JS source

### ISSUE THREAD

**Issue by LoicMahieu** [yeonjuan/html-eslint#218]
I would like to lint HTML code that is stored directly in like this:

```
const someView = `<div>Hello</div>`;
```

Prettier supports this by using a `html`  tag:

```
const someView = html`<div>Hello</div>`;
```

It works pretty well but eslint does not seems to support it.

**Comment by RobertAKARobin** [comment]
Duplicate of #196

**Comment by yeonjuan** [comment]
hello. @LoicMahieu  I'm happy to announce that this feature has been implemented and deployed in 0.29.0.
How to use
1. Updates both `@html-eslint/parser` and `@html-eslint/eslint-plugin` version to `0.29.0`.
```console
npm install -D @html-eslint/eslint-plugin@0.29.0 @html-eslint/parser@0.29.0
```

2. Edit your eslint configuration file

```js
import eslintHTML from "@html-eslint/eslint-plugin";
import eslintHTMLParser from "@html-eslint/parser";

export default [
  {
    "plugins": {
      "@html-eslint": eslintHTML
    },
    "rules": {
      // Specifies the @html-eslint rules to apply to Template Literal.
      "@html-eslint/no-inline-styles": 1,
      "@html-eslint/indent": 1,
     }
  },
  {
    "files": ["**/*.html"],
    "languageOptions": {
      parser: eslintHTMLParser,
    },
    "plugins": {
      "@html-eslint": eslintHTML,
    },
    "rules": {
      // Specifies the @html-eslint rule to apply to HTML files.
     "@html-eslint/no-inline-styles": 1
    }
},
]
```

3. Now you can see that this plugin linting the html syntax inside the template literal.

```js
// TaggedTemplateExpression
html`
<div style="${style}">
    ^^^^^^^^^^^^^^^// Unexpected usage of inline style

</div>`;

// TemplateLiteral
const code = /* html */`
<div style="${style}">
    ^^^^^^^^^^^^^^^// Unexpected usage of inline style

</div>`
```
4. If you want to specify that linting should be done with keywords other than `html`, you can utilize the settings option.

```js
 {
    "plugins": {
      "@html-eslint": eslintHTML
    },
    settings: {
        html: {
          templateLiterals: {
               // default options
               tags: ["^html$"],
               comments: ["^\\s*html\\s*$"],
          }
        }
    },
}
```
